src/calls-cost.js

```diff
@@ -39,6 +39,52 @@
  * @returns {CallsResponse}  - Processed information
 */
 
-function callsCost(calls) { }
+function callsCost(calls) {
+  let processedCalls = [];
+  const lCallCost = 0.2;
+  const nCallCost = 1.20;
+  const iCallCost = 7.56;
+  let totalCalls = 0;
+  let totalCost = 0;
+  let tempCost = 0
+  let callCost = 0
+  calls.forEach(val => {
```

Comment:
A `reduce` is perfect for this kind of thing. Whenever I see a bunch of `lets` that are being assigned later is kind of a red flag that a better pattern is out there

Here is a good video on reduce

 https://www.youtube.com/watch?v=PAwd4gjb7aI&t